collectors/python.d.plugin/mysql/mysql.chart.py

```diff
@@ -357,7 +359,7 @@ def slave_running(value):
         ]
     },
     'innodb_os_log': {
-        'options': [None, 'InnoDB OS Log Pending Operations', 'operations', 'innodb', 'mysql.innodb_os_log', 'line'],
+        'options': [None, 'InnoDB OS Log Pending Operations', 'operations/s', 'innodb', 'mysql.innodb_os_log', 'line'],
         'lines': [
             ['Innodb_os_log_pending_fsyncs', 'fsyncs', 'absolute'],
             ['Innodb_os_log_pending_writes', 'writes', 'absolute', -1, 1],
```

Comment:
absolutes are not `/s`, incrementals are. (netdata does interpolation only for incrementals)